.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[*.{py,ini}]
+indent_style = space
+indent_size = 4
```

Comment:
Are still  using `tox` within this repository, or are we replacing it with `uv`? If so, we should remove `.ini` from the _.editorconfig_ file, but we should add a section for `just` to it:

```suggestion
[{justfile,*.py}]
indent_style = space
indent_size = 4
```

Reply:
We're replacing it with uv; it makes dependency handling much easier.

Reply:
Cool by me. Let's just (heh) make sure `just` is in editorconfig. Some editors know what to do with justfile but I've found that some don't :sweat_smile: